### ISSUE TITLE
Installing BacSeq

### ISSUE THREAD

**Issue by alex-trist** [tmrcpsu/bacseq#1]
Hi!
I've been trying to install BacSeq, but haven't succeeded yet. The main issue is that conda takes forever to solve the environment, even at the first steps (MultiQC). Installing the programs separately (without conda) would affect BacSeq performance?
To restart the installation process, is it enough to remove the conda environment and run the install.sh again?
Thanks!

**Comment by hazmup** [comment]
Have you tried setting the channel priority setting of conda to flexible by `conda config --set channel_priority true`?

**Comment by alex-trist** [comment]
> Have you tried setting the channel priority setting of conda to flexible by `conda config --set channel_priority true`?

Thanks @hazmup. That solved the MultiQC issue, but now other conflicts appeared. 

I've tried installing python 3.9 in the environment nad works properly until the abricate step. But no annotations (prokka, eggnog, staramr) and no phylogenies are generated. 

Any thoughts?

**Comment by hazmup** [comment]
Are you installing python 3.9 manually? The env specification calls for version 3.7.

Is the rest of the software installing properly? What do the checks show when you run `source run_BacSeq.sh`?

**Comment by alex-trist** [comment]
Yes, I installed python 3.9 manually, because with python 3.7 I was only able to install abricate v0.4 (which takes different arguments and comes with different databases than the ones mentioned in abricate.sh).  
Running run_BacSeq.sh shows the paths for all programs; however, the pipeline itself crashes after abricate (no output for functional annotation, pangenome, phylogeny, etc.).
I am inspecting the log for answers, however I recently upgraded conda to 23.10, and was thinking of giving the new libmamba solver a try with a fresh install of the entire environment.
![Captura desde 2023-11-22 08-20-40](https://github.com/tmrcpsu/bacseq/assets/150893329/29ac1dc5-820c-45f0-9cf2-98b0371ccf81)

**Comment by hazmup** [comment]
I probably had the same problem and modified the `install.sh` file to use `mamba` instead of `conda` to solve the environments, and it solved it. I am now having some trouble running `bacseq`, specifically at the `eggnog-mapper` step, so I will get back to you when I figure this out as well.
In general, I am using `mamba` for everything and it's incredibly faster and just as reliable. Maybe `bacseq` should be modified to use this by default, and also to specify package versions explicitly.
